internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -624,6 +624,106 @@ func NewCoreContainer(cr *model.CryostatInstance, specs *ServiceSpecs, imageTag
 	authPropertiesPath := "/app/resources/io/cryostat/net/openshift/OpenShiftAuthManager.properties"
```

Comment:
This authPropertiesPath and probably some other things can also be cleaned up since they are no longer used in 3.0 and probably won't be reimplemented, or if they are then probably in quite a different form.

Reply:
Should we remove the properties file, but keep the ClusterRole [here](https://github.com/cryostatio/cryostat-operator/blob/0b380d227f7b047571f8adf0a2467281ac3a5758/api/v1beta2/cryostat_types.go#L480-L494)? The OAuth proxy could use that role for its authorization check, should the user wish to override the default.

Reply:
That makes sense. We can leave that in there for now and see if any adjustments are needed later when we get to deploying the OpenShift OAuth Proxy.